public_website/next.config.js

```diff
@@ -5,7 +5,6 @@ const nextConfig = {
   compiler: {
     styledComponents: true,
   },
-  pageExtensions: ['page.tsx', 'page.ts', 'page.jsx', 'page.js'],
```

Comment:
C'est ça qui ne fonctionne pas, et pourquoi les tests sont considéré comme des pages

Reply:
https://stackoverflow.com/questions/70832134/is-it-possible-to-put-test-files-under-pages-directory-in-next-js